According to the POSIX conventions each option name
should be a single alphanumeric character.

--- configure.orig	2024-04-20 02:07:27 UTC
+++ configure
@@ -71,7 +71,7 @@ has_debug=0
 force_absolute_dylib_install_names=0;
 ext_dll=so
 has_debug=0
-do_strip=--strip
+do_strip=-s
 while : ; do
     case "$1" in
         "")
